docker/env/Dockerfile

```diff
@@ -1,15 +1,25 @@
-FROM centos:7
+FROM python:3.8
 
-ADD install-prereqs.sh install-prereqs.sh
-RUN ./install-prereqs.sh docker
```

Comment:
It'll be good to update `docker/env/build_n_push.sh` too.

Reply:
to update what exactly ? I'm not sure I understood what do you mean here...

Reply:
You removed `install-prereqs.sh` from Dockerfile, but not from `build_n_push.sh`:  the latter script copies/removes `install-prereqs.sh` to run it in Dockerfile.

Reply:
now removed it